AboutPage.html

```diff
@@ -0,0 +1,71 @@
+                <a class="nav-link" href="#">Sign up/Log in</a>
+            </nav>
+                <a href="Index.html"><img class="logo" src="img\Updated logo.png"></a>
+        </section>
+    </div>
+
+    <img class="header-img" src="img\duy-pham-Cecb0_8Hx-o-unsplash.png">
+    <h1 class="about-header">Meet the team</h1>
+    
+    <div class="about-content">    
+        <section class="about-para">
+            <h2>Kyle Pace</h2>
+            <img class="para-img" src="img\circle-cropped (1).png">
+            <p>Full-stack web developer. I love coding and problem-solving. I work with HTML, CSS, and Javascript.</p>
+            <a href="https://github.com/kpace622" target="_blank"><i class="fab fa-github-square"></i></a>
+            <a href="https://twitter.com/PaceKyle" target="_blank"><i class="fab fa-twitter-square"></i></a>
+            <a href="https://www.linkedin.com/in/kyle-pace-a220b5156/" target="_blank"><i class="fab fa-linkedin"></i></a>
+        </section>
+
+        <section class="about-para">
+            <h2>Donald Coleman</h2>
+            <img class="para-img" src="img\circle-cropped.png">
+            <p>Full-Stack Web Developer that loves to code, debug, and help other learn.</p>
+            <br>
+            <a href="https://github.com/docoleman36" target="_blank"><i class="fab fa-github-square"></i></a>
+            <i class="fab fa-twitter-square"></i>
+            <a href="https://www.linkedin.com/in/djcoleman479/" target="_blank"><i class="fab fa-linkedin"></i></a>
+        </section>
+
+        <section class="about-para">
+            <h2>Jason Barr</h2>
+            <img class="para-img" src="img\circle-cropped (2).png">
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+                    eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                     Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+            <a href="https://github.com/jasonsbarr" target="_blank"><i class="fab fa-github-square"></i></a>
+            <a href="https://twitter.com/jasonsbarr" target="_blank"><i class="fab fa-twitter-square"></i></a>
+            <a href="https://www.linkedin.com/in/jasonsbarr/" target="_blank"><i class="fab fa-linkedin"></i></a>
+        </section>
+
+        <section class="about-para">
+            <h2>Shafi Masoumi</h2>
+            <img class="para-img" src="img\circle-cropped (5).png">
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+                    eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                     Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+            <a href="https://github.com/shafi2019" target="_blank"><i class="fab fa-github-square"></i></a>
+            <i class="fab fa-twitter-square"></i>
+        </section>
+    </div>
+
+   
+</body>
+
+</html>
```

Comment:
Needs newline at end of file - Github is complaining. See also other HTML files.